modules/dnn/src/layers/fast_convolution/fast_convolution.cpp

```diff
@@ -191,10 +194,7 @@ void runFastConv2d(InputArray _input, OutputArray _output, const Ptr<FastConv2d>
     }
 
 #if CV_NEON
-    if (conv->ifWinograd63
-         && inputShape[2] > 12 && inputShape[3] > 12
-         && inputShape[2] < 120 && inputShape[3] < 120
-         )
+    if (conv->ifWinograd63 && inputShape[2] > 12 && inputShape[3] > 12)
     {
```

Comment:
In the previous version, we set the maximum input size here. Because without this limitation, some tests like([ENet_accuracy](https://github.com/opencv/opencv/blob/4.x/modules/dnn/test/test_torch_importer.cpp#L440-L488)) would fail. With the new feature, we can remove this limitation.